composer.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "jeroen/mediawiki-github",
```

Comment:
Affirmative. Just wanted to note that it will be nice if this also gets fluffed as a follow-up.